AntennaTracker/Log.cpp

```diff
@@ -42,18 +42,44 @@ void Tracker::Log_Write_Vehicle_Baro(float pressure, float altitude)
         time_us         : AP_HAL::micros64(),
         press           : pressure,
         alt_diff        : altitude
-
     };
     DataFlash.WriteBlock(&pkt, sizeof(pkt));
 }
 
+struct PACKED log_Vehicle_Pos {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;
+    int32_t vehicle_lat;
+    int32_t vehicle_lng;
+    int32_t vehicle_alt;
+    float vehicle_heading;
+    float vehicle_speed;
+};
+
+// Write a vehicle pos packet
+void Tracker::Log_Write_Vehicle_Pos(int32_t lat,int32_t lng,int32_t alt,float heading,float ground_speed){
```

Comment:
Yes, I noticed these formatting issues as well and fixed it in a small follow-up commit.
https://github.com/ArduPilot/ardupilot/commit/0e3dfb9e5ef7bf1278bc7161072df2090fb599a0